### PR TITLE
OP-482 set date to end of day for comparison

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-482

Made the change and tested it with the example given in the JIRA issue

### CODE REVIEW

File: src/main/java/org/isf/utils/jobjects/DateTextField.java [hunk 1]
```diff
@@ -175,9 +176,12 @@ public GregorianCalendar getCompleteDate(){
...
 		}
-		date.set(GregorianCalendar.DAY_OF_MONTH,getDay());
-		date.set(GregorianCalendar.MONTH,getMonth()-1);
-		date.set(GregorianCalendar.YEAR,getYear());
+		date.set(Calendar.DAY_OF_MONTH,getDay());
+		date.set(Calendar.MONTH,getMonth()-1);
+		date.set(Calendar.YEAR,getYear());
+		date.set(Calendar.HOUR_OF_DAY, 23);
+		date.set(Calendar.MINUTE, 59);
+		date.set(Calendar.SECOND, 59);
```
Comment (mwithi) [inline src/main/java/org/isf/utils/jobjects/DateTextField.java:184]: As said in comments on OP-482, this is accettable as temporary solution, could we add a comment for this? Thanks!

**Review by mwithi — APPROVED** [review]
LTGM, only one comment required

File: src/main/java/org/isf/utils/jobjects/DateTextField.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/jobjects/DateTextField.java]: @mwithi FWIW,  I added the comment.

**Review by mwithi — APPROVED** [review]
Thanks a lot!

**Pull request merged**